src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1669,7 +1669,7 @@ private TaskDialogEventArgs DisplayEngineFailureMessage(Exception exception)
 
             string description = (exception is HeapCorruptionException)
                 ? exception.Message
-                : @Properties.Resources.ExceptionIsNotHeapCorruptionDescription;
+                : @Properties.Resources.DisplayEngineFailureMessageDescription;
 
             const string imageUri = "/DynamoCoreWpf;component/UI/Images/task_dialog_crash.png";
             var args = new TaskDialogEventArgs(
```

Comment:
Ouch, a `DynamoCoreWpf` reference in `DynamoModel`! I have done the following:

- Removed `DynamoCoreWpf` strings from `DynamoCore` project [in this pull request](https://github.com/DynamoDS/Dynamo/pull/4398).
- Added [a new task](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7308) for refactoring `TaskDialogEventArgs`.

Please help to review the above PR :)